controller/Lists.php

```diff
@@ -71,32 +70,27 @@ class Lists extends tao_actions_CommonModule
     private $isListsDependencyEnabled;
 
     /**
-     * Show the list of users
+     * Show the list
      * @return void
      */
     public function index()
     {
         tao_helpers_Scriptloader::addCssFile(Template::css('lists.css', 'tao'));
         $this->defaultData();
 
-        $myAdderFormContainer = new tao_actions_form_List();
-        $myForm = $myAdderFormContainer->getForm();
-
-        if ($myForm->isSubmited()) {
-            if ($myForm->isValid()) {
-                $values = $myForm->getValues();
-                $newList = $this->getListService()->createList($values['label']);
-                $i = 0;
-                while ($i < $values['size']) {
-                    $this->getListService()->createListElement($newList, __('element') . ' ' . ($i + 1));
-                    $i++;
-                }
+        $newListFormContainer = new CreateListForm();
```

Comment:
Do we need this form in the BE? I think the FE can do everything now. Maybe this class is not necessary.

Reply:
I've removed this class and placed the form HTML directly on the template, so I'm marking this thread as solved.

Reply:
I'm finding it difficult to make this work without relying on this class extending `tao_helpers_form_FormContainer` (as the former `tao_actions_form_List` does). For the time being, I've restored the former approach of using that instead of having the HTML for the "Create List" `<form>` moved to the template.

Reply:
This is not true anymore: The form HTML is now in the template (and `CreateListForm` has been), so I'm marking this thread as solved.